### DISCUSSION TITLE
can't see traffic in kiali

### DISCUSSION THREAD

**Discussion by saarwasserman** [kiali/kiali#4177] [Q&A] [answered]
Hi,

I am not getting any service traffic in kiali.
kiali version: 1.34.0
istio:  1.6.8
prometheus: 2.26.0

1. kiali was able to fetch prometheus version and I don't see any errors in logs related to prometheus.
2. when I query prometheus for istio_requests_total with the same query I get from kiali logs I get data back

is there a minimum value in order to show traffic?
what can be the cause?
I will appreciate any help.

**Comment by lucasponce** [comment]
I'm not sure, take note that Istio 1.6.8 was tested against these combination of addons:
https://github.com/istio/istio/tree/1.6.14/samples/addons

So, a change of the version may result in some unexpected new problem.

What you report sounds familiar to the problem of other user that was due a newer version of prometheus, if you give a try with the version used in the addons, you'd get any difference ?

**Reply by saarwasserman** [reply]
i'll try and update thanks.

**Reply by saarwasserman** [reply]
it works with istio 1.8.6. with same prometheus version.
However, It might as well be different configs of prometheus (or istio). I used the prometheus addon now. (with same 2.26.0 version). although I saw istio data with previous prometheus setup.

**Reply by saarwasserman** [reply]
there are differences in setup and versions probably. had some issues with graph later on which got stuck on old labels. switched back to 1.18 using the addon

**Comment by xeviknal** [comment]
+1 to what Lucas said.
However, I see that there is a patch release for 1.34. You are using the
1.34.0, you might want to use the 1.34.1. Team usually does patch releases
when there are big issues.
So I'd give a try to v1.34.1 (or the same: v1.34 that falls back to 1.34.1)

On Wed, Jul 14, 2021 at 11:36 AM saarwasserman ***@***.***>
wrote:

> i'll try and update thanks.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kiali/kiali/discussions/4177#discussioncomment-1003011>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEV3NR57ZU6MTJVE5GEZ5DTXVLB3ANCNFSM5ALB4D3A>
> .
>


-- 
Xavier Canal i Masjuan

**Comment by saarwasserman** [comment, marked as answer]
my fault. prometheus urll. working on with kiali 1.34.1
